### PR TITLE
Rails api 2

### PR DESCRIPTION
**Pull request by lcoq**

### CODE REVIEW

File: rails/Gemfile [hunk 1]
```diff
@@ -13,4 +14,10 @@ group :development, :test do
   # Access an IRB console on exception pages or by using <%= console %> in views
   gem 'web-console', '~> 2.0'
   gem 'spring'
+
+  gem 'database_cleaner'
+  gem 'factory_girl_rails'
+  gem 'minitest-rails', github: 'bobbus/minitest-rails', branch: 'fix-name-error'
```
Comment (lcoq) [inline rails/Gemfile]: not sure if we need to use this fork

**Comment by lcoq** [comment]
At this time local environment needs to change the `protect_from_forgery` to `:null_session`

File: rails/Gemfile (hunk 1 above)
Comment (bobbus) [inline rails/Gemfile]: If not necessary no

File: rails/bin/_guard-core [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
```
Comment (bobbus) [inline rails/bin/_guard-core]: Is this file necessary?

**Comment by bobbus** [comment]
Reviewed
